lib/profile/icon_button.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+ 
+class ProfileIconButton extends StatelessWidget {
+  final IconData name;
+  final String caption;
+  ProfileIconButton({Key key, @required this.name, @required this.caption}) : super(key: key);
+  
+  @override
+    Widget build(BuildContext context) {
+      return Container(
+        margin: const EdgeInsets.all(10.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            Container(
+              margin: const EdgeInsets.only(bottom: 2.0),
+              child: Material(
+                color: Colors.white,
```

Comment:
In case there's more meaning to why `Material` is being used, you can set its color to `Colors.transparent`. That way until we tackle the responsive design issue, we don't have to worry about the buttons looking out of place on different screen sizes.

> <img src="https://user-images.githubusercontent.com/23146829/82391106-b3c48d80-99fd-11ea-94d6-b00c857a1251.png" alt="A screenshot of the user profile page with a red box outlining the icon buttons that have a white background" width=200 />  -> <img src="https://user-images.githubusercontent.com/23146829/82391178-e7071c80-99fd-11ea-9f08-3e16e2956b54.png"  alt="A screenshot of the user profile page with a red box outlining the icon buttons that have a transparent background now" width=200 />

Reply:
Nice catch!!

Reply:
Also, I am using `Material` since `Ink` classes need to be under `Material` widgets! See: https://api.flutter.dev/flutter/material/Ink-class.html

Reply:
Thanks!